db.go

```diff
@@ -150,27 +163,27 @@ func (db *DB) DropAll(ctx context.Context) error {
 }
 
 func (db *DB) DropData(ctx context.Context) error {
-	return db.gxy.DropData(ctx)
+	return db.defaultNamespace.DropData(ctx)
 }
 
 func (db *DB) AlterSchema(ctx context.Context, sch string) error {
-	return db.gxy.AlterSchema(ctx, sch)
+	return db.defaultNamespace.AlterSchema(ctx, sch)
 }
 
 func (db *DB) Query(ctx context.Context, q string) (*api.Response, error) {
-	return db.gxy.Query(ctx, q)
+	return db.defaultNamespace.Query(ctx, q)
 }
 
 func (db *DB) Mutate(ctx context.Context, ms []*api.Mutation) (map[string]uint64, error) {
-	return db.gxy.Mutate(ctx, ms)
+	return db.defaultNamespace.Mutate(ctx, ms)
 }
 
 func (db *DB) Load(ctx context.Context, schemaPath, dataPath string) error {
```

Comment:
Let's just make this one `LoadSchema(ctx, schemaPath)`, since we already have `LoadData` that does the same thing with `dataPath`.
```suggestion
func (db *DB) Load(ctx context.Context, schemaPath, dataPath string) error {
```